specifications/api/swagger.yaml

```diff
@@ -2389,13 +2412,17 @@ components:
         - inputs
         - outputs
         - withdrawals
+        - assets_minted
+        - assets_burnt
       properties:
         id: *transactionId
         fee: *amount
         inputs: *ApiInputsGeneral
         outputs: *ApiOutputsGeneral
         collateral: *ApiInputsGeneral
         withdrawals: *ApiWithdrawalsGeneral
+        assets_minted: *ApiMintedBurntAssets
+        assets_burnt: *ApiMintedBurntAssets
```

Comment:
Hi @paweljakubas 

I'm wondering why are there two of these fields, when both of these fields are `ApiMintedBurnedAssets`?

Shouldn't it be something like:
```
        assets_minted: *ApiMintedAssets
        assets_burned: *ApiBurnedAssets
```

Reply:
adopted